### ISSUE TITLE
Windows' Custom Scaling Issue with Fullscreen

### ISSUE THREAD

**Issue by UmbrellaCorporat** [RPCS3/rpcs3#14180]
### Quick summary

Rpcs3 does not go full screen when using Windows' custom scaling. Entering a custom scaling size between 100% and 500% for a smaller scale size on a larger TV, like 190, prevents rpcs3 from going fully full screen. You will still see the taskbar and a small glow of your desktop wallpaper on the right side of the screen. However, when turning off Windows custom scaling, rpcs3 goes full screen without any issues.

### Details

When using custom scaling settings between 100% and 500% on a Windows computer, rpcs3 does not enter fullscreen mode properly. Instead, even when attempting to go fullscreen, the taskbar remains visible at the bottom of the screen, and a small portion of the desktop wallpaper is still visible on the right side of the screen. This can be visually distracting and takes away from the immersive experience of playing games in fullscreen mode.

To put it simply, when you enable fullscreen mode in rpcs3 with custom scaling enabled, the game window does not completely occupy the entire screen as it should. This is in contrast to the expected behavior, where the game should cover the entire display without any elements from the operating system, such as the taskbar or desktop wallpaper, being visible.

However, when you disable the custom scaling in the Windows settings, rpcs3 behaves correctly and enters fullscreen mode without any issues. The game window fills up the entire screen as expected, ensuring an uninterrupted gaming experience.

It's important to note that this issue is specific to rpcs3 and its interaction with Windows' custom scaling feature. Other applications and games may not be affected in the same way, so it appears to be a compatibility issue between rpcs3 and the custom scaling functionality of Windows.
[RPCS3.log.gz](https://github.com/RPCS3/rpcs3/files/12005922/RPCS3.log.gz)

### Attach a log file

[RPCS3.log.gz](https://github.com/RPCS3/rpcs3/files/12005976/RPCS3.log.gz)

### Attach capture files for visual issues

[https://drive.google.com/file/d/1_8tB96upcqyarhx-iMnt9dUIR2QQrbos/view?usp=sharing](url)

### System configuration

Ryzen 7 3700X processor
32GB of dual-channel RAM running at 3,200 MHz
An original model MSI GeForce RTX 2070 graphics card with 8GB of VRAM.
Windows 11

**Comment by Megamouse** [comment]
That looks like a windows bug.
For instance, the "hide taskbar automatically" feature barely works on 190%

**Comment by UmbrellaCorporat** [comment]
Yes, I tried hiding the taskbar, and it does hide the taskbar automatically as stated. However, you may still encounter glowing issues that can appear from the bottom or sides of your screen, with your Windows desktop gleaming out. It's just distracting, and I find it odd that it happens. I couldn't figure it out. I mean, I went through everything trying to solve it, but it turns out that it's some kind of issue with Windows custom scaling. I tested everything else out, like Steam, Epic, GOG, DuckStation, RetroArch, PCSX2, etc. They all work as expected in fullscreen mode.

**Comment by Asinin3** [comment]
Does it still happen with https://github.com/RPCS3/rpcs3/pull/12471

**Comment by Megamouse** [comment]
Yes

**Comment by digant73** [comment]
is that still an issue?

**Comment by LoweysLitsman** [comment]
Can be closed, works fine on both Exclusive Fullscreen and Borderless Fullscreen w/ custom scaling set to 190%.
Tested on Win11 w/ RTX 3080

![Image](https://github.com/user-attachments/assets/3f4b380a-bc72-423e-84bb-c96eee360ceb)
